src/polyswarmd/config.py

```diff
@@ -32,9 +32,15 @@
 }
 
 
-def is_service_reachable(session, uri):
-    r = session.get(uri)
-
+def is_service_reachable(session, uri, is_ethereum=False):
+    if is_ethereum:
```

Comment:
`is_ethereum` is True when checking whether the Ethereum service (either Geth or Parity) is available, correct?

If so, would this break using Geth or does Geth also support the same `POST` message that Parity does?

If they both support the same `POST` message, it's probably best to standardize on the `POST` message, and this should be merged into mainline.

Reply:
It does support not only parity but also geth since geth can accept POST as well. So I think this can be merged soon.

Reply:
Just tested this with get. Able to get status correctly.